packages/@react-native-windows/cli/src/e2etest/autolink.test.ts

```diff
@@ -528,6 +534,25 @@ test('Indirect autolink dependency', () => {
   );
 });
 
+// eslint-disable-next-line complexity
```

Comment:
Remove?

Reply:
same below

Reply:
It's not technically necessary here for autolink, but we have too many options in run-windows

Reply:
Is it possible change the code in run-windows so that it doesn't omit the warning, instead of disabling it for both?

Reply:
fixed